MSOfficeUpdates/MSOfficeMacURLandUpdateInfoProvider.py

```diff
@@ -337,6 +338,16 @@ def get_installer_info(self):
             or "10.10.5"
         )
 
+        # Make sure that the minimum_os_version is at least higher than the pre defined value
+        if not version_equal_or_greater(
+            pkginfo["minimum_os_version"],
+            PROD_DICT[self.env["product"]].get("minimum_os", "10.10.5")
```

Comment:
Where is this magic "10.10.5" value coming from?

Reply:
Other than that unexplained "default" value of "10.10.5", this seems fine, but will lead to more maintenance of MSOfficeMacURLandUpdateInfoProvider.py. Unavoidable, perhaps.

Reply:
Sorry, @gregneagle, I have missed your comments. I have chosen "10.10.5" as this is the default value for `minimum_os_version` as defined in lines 335-339.